### DISCUSSION TITLE
Defining keyboard shortcuts

### DISCUSSION THREAD

**Discussion by Zft0** [nroduit/Weasis#286] [Ideas]
Weasis is an amazing and wonderful viewer and I would like to thank you very much Nicolas.
I just have a suggestion that it would be great if we can define our own shortcuts (e.g. pressing Ctrl+Shift+u results in the dialogue box that appears after clicking on File->Open->Image).

**Comment by nroduit** [comment]
Thanks for your feedback. This is already on my to-do list but not with a high priority.
